.gitignore

```diff
@@ -13,4 +13,5 @@ npm-debug.log
 app/bower_components/
 bower_components/
 dist/
-build/
\ No newline at end of file
+build/
+docs/
```

Comment:
I added `docs/` here since there will probably be a lot of churn while in development. I am fine with adding the docs to version control as well.